docs/meson.build

```
@@ -31,3 +32,13 @@ custom_target(
   install_dir: join_paths(datadir, 'doc', 'rauc'),
   install: get_option('htmldocs'),
 )
+
+custom_target(
+  'manpages',
+  output: ['rauc.1'],
+  depend_files: sources_doc,
+  command: [sphinx, '-b', 'man', meson.current_source_dir(), '@OUTDIR@'],
+  build_by_default: get_option('manpages'),
+  install: true,
```

```suggestion
  install: get_option('manpages'),
```

Required as we otherwise attempt to install a file we did not generate.

---

I get `File 'docs/rauc.1' not found, skipping`, which seems non-fatal. I've changed it to make it cleaner.